static/src/stylesheets/base/_iframe.scss

```diff
@@ -0,0 +1,6 @@
+/* General styles applied when the window is embedded inside another document via iframe */
```

Comment:
@sndrs @SiAdcock these styles are only meant for atoms that are embedded in iframes... let me know if you'd rather this file be elsewhere